src/linear_algebra/multilinear.lean

```diff
@@ -80,8 +80,27 @@ variables [semiring R]
 
 instance : has_coe_to_fun (multilinear_map R M₁ M₂) := ⟨_, to_fun⟩
 
+initialize_simps_projections multilinear_map (to_fun → apply)
+
+@[simp] lemma to_fun_eq_coe : f.to_fun = f := rfl
+
+@[simp] lemma coe_mk (f : (Π i, M₁ i) → M₂) (h₁ h₂ ) :
+  ⇑(⟨f, h₁, h₂⟩ : multilinear_map R M₁ M₂) = f := rfl
+
+theorem congr_fun {f g : multilinear_map R M₁ M₂} (h : f = g) (x : Π i, M₁ i) : f x = g x :=
+congr_arg (λ h : multilinear_map R M₁ M₂, h x) h
+
+theorem congr_arg (f : multilinear_map R M₁ M₂) {x y : Π i, M₁ i} (h : x = y) : f x = f y :=
+congr_arg (λ x : Π i, M₁ i, f x) h
+
+theorem coe_inj ⦃f g : multilinear_map R M₁ M₂⦄ (h : ⇑f = g) : f = g :=
+by cases f; cases g; cases h; refl
```

Comment:
Following #3071:
```suggestion
theorem coe_injective ⦃f g : multilinear_map R M₁ M₂⦄ (h : ⇑f = g) : f = g :=
by cases f; cases g; cases h; refl
```

Reply:
This is to match `ring_hom.coe_inj`. The conclusion doesn't use `injective`, so surely the name shouldn't either?

Reply:
So I think my stance is:

* It would be great (but out of scope) if more of the `_inj` lemmas could be expressed using `injective`.
* Until they are, `_inj` is the correct name, since dot notation works only on the `_injective` ones.

Reply:
Ah, of course. Maybe the best way to fit it with mathlib style is to make it an `inj_iff` instead?

Reply:
Mathlib style is the way [I already have it](https://leanprover-community.github.io/mathlib_docs/find/coe_inj): `one_hom.coe_inj`, `mul_hom.coe_inj`, `monoid_hom.coe_inj`, ... all have this naming convention.